data/systemd/snapd.failure.service.in

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Failure handling of the snapd snap
+
+[Service]
+Type=oneshot
+ExecStart=@libexecdir@/snapd/snap-failure run
+RemainAfterExit=true
```

Comment:
Why `RemainAfterExit`?

Reply:
Mostly to make it easier to inspect in tests etc.